docs/reference/migration/migrate_2_0.asciidoc

```diff
@@ -400,3 +400,17 @@ The obsolete parameters `expand_wildcards_open` and `expand_wildcards_close` are
 supported by the snapshot and restore operations. These parameters have been replaced by
 a single `expand_wildcards` parameter. See <<multi-index,the multi-index docs>> for more.
 
+=== Multiple data.path striping
+
+In previous versions the multiple data locations allows to stripe it. The striping is simple,
+placing whole files in one of the locations, and deciding where to place
+the file based on the value of the `index.store.distributor` setting. Yet, in elasticsearch 2.0
+The stripe option is no supported anymore since it can cause the loss of all data on a node even 
+if a single disk is lost. It also caused obscure failures since shards might detect the disk loss
+very late in the process causing potential dataloss even across replicas of a shard.
+
+As of 2.0 each shard is allocated on a single disk rather than being striped across data paths. Yet,
+elasticsearch will upgrade shards into a single data location on first node startup. In the case where
+there is not enough disk space to upgrade the node the upgarde is canceled and can only be resumed onces
+enough disk space is abvailable to copy the shard to upgrade. 
+
```

Comment:
=== Multiple data.path striping

    Previously, if the `data.path` setting listed multiple data paths, then a
    shard would be ``striped'' across all paths by writing a whole file to each
    path in turn (in accordance with the `index.store.distributor` setting).  The
    result was that the files from a single segment in a shard could be spread
    across multiple disks, and the failure of any one disk could corrupt multiple
    shards.

    This striping is no longer supported.  Instead, different shards may be
    allocated to different paths, but all of the files in a single shard will be
    written to the same path.

    If striping is detected while starting Elasticsearch 2.0.0 or later, all of
    the files belonging to the same shard will be migrated to the same path. If
    there is not enough disk space to complete this migration, the upgrade will be
    cancelled and can only be resumed once enough disk space is made available.